pulsar-client/src/main/java/org/apache/pulsar/client/impl/conf/ClientConfigurationData.java

```diff
@@ -51,7 +51,8 @@
     private String tlsTrustCertsFilePath = "";
     private boolean tlsAllowInsecureConnection = false;
     private boolean tlsHostnameVerificationEnable = false;
-    private int concurrentLookupRequest = 50000;
+    private int concurrentLookupRequest = 5000;
+    private int maxLookupRequest = 20000;
```

Comment:
based the logic you changed, we need to set `maxLookupRequest` to 50000 to make the logic similar as before, right?

before this change, `concurrentLookupRequest` is 50000